code/__HELPERS/events.dm

```diff
@@ -0,0 +1,52 @@
+#define UNLIT_AREA_BRIGHTNESS 0.2
+
+/**
+ * Finds us a generic maintenance spawn location.
+ *
+ * Goes through the list of the generic mainteance landmark locations, checking for atmos safety if required, and returns
+ * a valid turf. Returns MAP_ERROR if no valid locations are present.
+ * Returns nothing and alerts admins if no valid points are found. Keep this in mind
+ * when using this helper.
+ */
+
+/proc/find_maintenance_spawn(atmos_sensitive = FALSE, require_darkness = FALSE)
+	var/list/possible_spawns = list()
+	for(var/spawn_location in GLOB.generic_maintenance_landmarks)
```

Comment:
could we type this var and do `as anything` or are there a bunch of different types in this global list?

Reply:
do not use `as anything` if it has a bunch of different types and you're going to typecheck it anyways. if you aren't going to type it or typecheck it- doesn't matter i don't think